PowerPointLabs/PowerPointLabs/Models/PowerPointCurrentPresentationInfo.cs

```diff
@@ -57,5 +63,35 @@ public static IEnumerable<PowerPointSlide> SelectedSlides
                 return slides;
             }
         }
+
+        private static bool ActivateWindow() 

```

Comment:
I assume that this method checks if there is an active window right? Perhaps the name can be changed to IsWindowActive or HasActiveWindow to make the naming more clear.

Reply:
It tries to activate the first window at Line 74. Thus it does what the function says, which is to try to activate a window. However, I am unable to confirm that this will work because it is difficult to replicate the situation that Prof is facing.